lib/modules/showImages.js

```diff
@@ -1185,6 +1210,14 @@ function generateMedia(options: ExpandoMedia, context: {| href: string |}) {
 			throw new Error(`Unreachable: invalid media type ${options.type}`);
 	}
 
+	if (options.crosspostData) {
```

Comment:
I think this is a better fit in [`generateMedia` in `retrieveExpandoOptions`](https://github.com/honestbleeps/Reddit-Enhancement-Suite/blob/c65e3cc6e66d14d96efaafcef48f4aff36376c50/lib/modules/showImages.js#L1070), alongside with adding the site attribution. That way `crosspostData` can be accessed directly from `thing` (and not stored as `crosspostData`).